app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def increment_counter
+    current_post = Post.find(post_id)
+    current_post.update(likes_counter: (current_post.likes_counter || 0) + 1)
+  end
+end
```

Comment:
- [ ] [OPTIONAL] Like my previous suggestion, it would be nice to use the `after_save` Rails Active Callback here 👍

Reply:
Thanks for suggestions 👏 make the necessary adjustment